websites/R/RealLifeGTAV/presence.ts

```diff
@@ -11,115 +11,54 @@ presence.on("UpdateData", async function () {
-    }
-  }
   if (
-    document.location.hostname == "reallifegtav.de" ||
-    document.location.hostname == "rlv.link"
+    document.location.hostname === "reallifegtav.de" ||
+    document.location.hostname === "rlv.link"
   ) {
     if (set_timeElapsed) {
       presenceData.startTimestamp = browsingStamp;
     }
     if (
-      (urlpath[1] == "" || document.location.pathname.includes("/home")) &&
+      (urlpath[1] === "" || document.location.pathname.includes("/home")) &&
       urlpath[2] != ""
     ) {
-      presenceData.smallImageKey = "downloading";
       presenceData.details =
-        "Startseite" +
-        ifSettingEnabled(
-          set_showUsername,
-          " | User: " + document.querySelector("#RPC_Username").textContent
-        );
+        "Startseite" + if(set_showUsername) " | User: " + document.querySelector("#RPC_Username").textContent;
     } else if (document.location.pathname.includes("/login")) {
       presenceData.details =
-        "Meldet sich an" +
-        ifSettingEnabled(
-          set_showUsername,
-          " | User: " + document.querySelector("#RPC_Username").textContent
-        );
-      presenceData.smallImageKey = "writing";
+        "Meldet sich an" + if(set_showUsername) " | User: " + document.querySelector("#RPC_Username").textContent;
     } else if (document.location.pathname.includes("/register")) {
       presenceData.details =
-        "Registriert sich" +
-        ifSettingEnabled(
-          set_showUsername,
-          " | User: " + document.querySelector("#RPC_Username").textContent
-        );
-      presenceData.smallImageKey = "writing";
+        "Registriert sich" + if(set_showUsername) " | User: " + document.querySelector("#RPC_Username").textContent;
     } else if (document.location.pathname.includes("/dashboard")) {
       presenceData.details =
-        "Dashboard" +
-        ifSettingEnabled(
-          set_showUsername,
-          " | User: " + document.querySelector("#RPC_Username").textContent
-        );
-      presenceData.smallImageKey = "reading";
+        "Dashboard" + if(set_showUsername) " | User: " + document.querySelector("#RPC_Username").textContent;
     } else if (document.location.pathname.includes("/faq")) {
       presenceData.details =
-        "FAQ" +
-        ifSettingEnabled(
-          set_showUsername,
-          " | User: " + document.querySelector("#RPC_Username").textContent
-        );
-      presenceData.smallImageKey = "reading";
+        "FAQ" + if(set_showUsername) " | User: " + document.querySelector("#RPC_Username").textContent;
     } else if (document.location.pathname.includes("/jobs")) {
       presenceData.details =
-        "Jobs" +
-        ifSettingEnabled(
-          set_showUsername,
-          " | User: " + document.querySelector("#RPC_Username").textContent
-        );
-      presenceData.smallImageKey = "writing";
+        "Jobs" + if(set_showUsername) " | User: " + document.querySelector("#RPC_Username").textContent;
```

Comment:
Line 43 and other related lines with `if()`, I never once said to change to an if statement. Use. A. Ternary. Operator. It lets you define a variable based on if it is true or false, which would be used in your case. Also good for Code Optimisation and shortening.